fappdeploy/httptrigger/GHubEventDispatch.cs

```diff
@@ -28,40 +28,84 @@
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Extensions.EventGrid;
+using System.Collections.Generic;
 
 public static class GridEventHandler{
-    [FunctionName("PrettyPoisons")]
+
+    private static string ParseEventGridValidationCode(dynamic requestObject)
+    {
+        var webhook_res = string.Empty;
+        if (requestObject != null && requestObject[0]["data"] != null){
+            var validationCode = requestObject[0].data.validationCode;
+            if(validationCode != null){
+                webhook_res = Newtonsoft.Json.JsonConvert.SerializeObject(new Newtonsoft.Json.Linq.JObject {["validationResponse"]= validationCode});
+            }
+        }
+        return webhook_res;
+    }
+    
+    private static string ParseMachineLearningEvent(dynamic requestObject)
+    {
+        return requestObject[0]["data"];
+    }
+    
+    private static string ParseBlobStorageEvent(dynamic requestObject)
+    {
+        return requestObject[0]["data"];
+    }
+    
+    private static string ParseContainerRegistryEvent(dynamic requestObject)
+    {
+        return requestObject[0]["data"];
+    }
+    
+    [FunctionName("generic_triggers")]
```

Comment:
this change will also require a change in the ARM templates... we might want to think about that